dev-libs/boost/boost-1.88.0.ebuild

```diff
@@ -207,6 +205,108 @@ multilib_src_compile() {
 	fi
 }
 
+# TODO: multilib-minimal_abi_src_test is somehow not invoked
+src_test() {
+	# only use the best ABI
+	BUILD_DIR="${S}-$(echo $(multilib_get_enabled_abi_pairs) | awk '{print $NF}')"
+
+	# create custom options for tests based on the build settings
+	TEST_OPTIONS=("${OPTIONS[@]}")
+
+	# dial down log output - the full b2 command used to compile & run
+	# a test suite will be printed by ejam and can be used to build
+	# and run the tests in a test suite's directory.
+	TEST_OPTIONS=("${TEST_OPTIONS[@]/-d+2/-d0}")
+
+	# The following libraries do not compile or fail their tests:
+	# algorithm: fails to use std::reverse_copy (std=c++20?)
```

Comment:
I just realized that I can add comments into arrays on separate lines, so I'm considering moving the failures down to their respective elements. Would this be better or worse for readability?

Reply:
IMO better to move them into the array, yes.

Reply:
You could probably also remove the prefix from the comment as it's redundant with the same name repeated on the next line as the array element. No real difference except shorter lines == sometimes easier to read.

Reply:
Makes sense - cleaned up further, thanks.